recipe-client-addon/test/browser/browser_about_studies.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+Cu.import("resource://testing-common/ContentTask.jsm", this);
```

Comment:
You don't need to manually import this. It's automatically available on test scopes, like BrowserTestUtils and friends.